### PR TITLE
fix(setup-wallet): titles and hw wallet prod

### PR DESCRIPTION
**Pull request by banklesss**
Related to:

- [YOMO-1305](https://emurgo.atlassian.net/browse/YOMO-1305)
- [YOMO-1298](https://emurgo.atlassian.net/browse/YOMO-1298)
- [YOMO-1297](https://emurgo.atlassian.net/browse/YOMO-1297)

[YOMO-1305]: https://emurgo.atlassian.net/browse/YOMO-1305?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[YOMO-1298]: https://emurgo.atlassian.net/browse/YOMO-1298?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[YOMO-1297]: https://emurgo.atlassian.net/browse/YOMO-1297?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by michalszorad — APPROVED** [review]

File: apps/wallet-mobile/src/features/SetupWallet/useCases/ChooseSetupType/ChooseSetupTypeScreen.tsx [hunk 1]
```diff
@@ -42,19 +42,19 @@ export const ChooseSetupTypeScreen = () => {
...
     }
 
-    navigation.navigate('setup-wallet-choose-network')
+    navigation.navigate('setup-wallet-create-choose-network')
   }
 
   const handleRestore = () => {
     setUpTypeChanged('restore')
 
     if (isProduction()) {
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/SetupWallet/useCases/ChooseSetupType/ChooseSetupTypeScreen.tsx:52]: can we have a comment here, "On production the step of network is skipped" the first thing I think when I read it, is that a dev test was left behind ;)

File: apps/wallet-mobile/src/features/SetupWallet/useCases/ChooseSetupType/ChooseSetupTypeScreen.tsx [hunk 2]
```diff
@@ -64,14 +64,8 @@ export const ChooseSetupTypeScreen = () => {
...
     setUpTypeChanged('hw')
-
-    if (isProduction()) {
-      networkIdChanged(HASKELL_SHELLEY.NETWORK_ID)
-      navigation.navigate('setup-wallet-check-nano-x')
-      return
-    }
-
-    navigation.navigate('setup-wallet-choose-network')
+    networkIdChanged(HASKELL_SHELLEY.NETWORK_ID)
```
Comment (stackchain) [inline apps/wallet-mobile/src/features/SetupWallet/useCases/ChooseSetupType/ChooseSetupTypeScreen.tsx]: why for HW is fixed network?

**Comment by stackchain** [comment]
waiting for the comments to merge @banklesss

File: apps/wallet-mobile/src/features/SetupWallet/useCases/ChooseSetupType/ChooseSetupTypeScreen.tsx (hunk 2 above)
Comment (banklesss) [reply apps/wallet-mobile/src/features/SetupWallet/useCases/ChooseSetupType/ChooseSetupTypeScreen.tsx]: original ticket has been declined

**Comment by banklesss** [comment]
> waiting for the comments to merge @banklesss

done

**Review by stackchain — APPROVED** [review]

**Pull request merged**